nixos/tests/certmgr.nix

```diff
@@ -94,7 +109,7 @@ let
               serviceConfig = {
                 User = "cfssl";
                 Type = "oneshot";
-                WorkingDirectory = config.services.cfssl.dataDir;
+                inherit (config.systemd.services.cfssl.serviceConfig) WorkingDirectory StateDirectory StateDirectoryMode;
```

Comment:
`cfssl`'s home directory was being created by systemd via `StateDirectory`. Since `cfssl-init` runs before `cfssl`, the directory hasn't been created yet so we inherit the values so it should do the same thing.